### PR TITLE
more info in README

### PR DESCRIPTION
**Pull request by varunon9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
 ## sell-my-services
 
-We help people to offer their services to others easily and quickly. SellMyServices is a platform where a seller can publish his
-services. These services then can be searched by seekers in their nearby locations. 
+An open source content search tool. This is the backend repository of https://sellmyservices.in.
```
Comment (AnshulMalik) [inline README.md]: content search tool?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
-4. Seekers search services on our platform.
-5. SellMyServices list all the nearby sellers who are offering services searched by seeker. 
-6. Seeker then can browser seller's services, their ratings, location etc and if interested can directly chat to sellers.
-7. After availing a service, a seeker can share feedback and rating to corresponding seller on our platform.
+#### The core idea
+
+SellMyServices is a platform to match `service seekers` to `service sellers`. A normal flow is-
+
+1. User installs our [Android app](https://play.google.com/store/apps/details?id=me.varunon9.sellmyservices)
+2. From the MainActivity he searches for services of his interset. Let's say he is looking for a `freelancer`.
```
Comment (AnshulMalik) [inline README.md]: from MainActivity?

File: README.md (hunk 1 above)
Comment (varunon9) [reply README.md]: it's the general term for this product. This idea can be made specific e.g. nearby restaurant search tool etc.

File: README.md (hunk 2 above)
Comment (varunon9) [reply README.md]: I'll update this to MainScreen

Comment (AnshulMalik) [reply README.md]: We can just remove screen, and `interest`

Comment (varunon9) [reply README.md]: 2. In the app he searches for some services. Let's say he is looking for a `freelancer`. ??

Comment (AnshulMalik) [reply README.md]: Sounds good.

File: README.md (hunk 1 above)
Comment (AnshulMalik) [reply README.md]: Content search seems too general to me

Comment (varunon9) [reply README.md]: local services search tool?

Comment (varunon9) [reply README.md]: updated. Lets merge this :)

**Pull request merged**